butterfree/hooks/schema_compatibility/cassandra_table_schema_compatibility_hook.py

```diff
@@ -0,0 +1,59 @@
+"""Spark table schema compatibility Hook definition."""
+
+from pyspark.sql import DataFrame
+
+from butterfree.clients import CassandraClient
+from butterfree.constants import DataType
+from butterfree.hooks.hook import Hook
+
+
+class CassandraTableSchemaCompatibilityHook(Hook):
+    """Hook to verify the schema compatibility with a Cassandra's table.
+
+    Verifies if all columns presented on the dataframe exists and are the same
+    type on the target Cassandra's table.
+
+    Attributes:
+        cassandra_client: client to connect to Cassandra DB.
+        table: table name.
+
+    """
+
+    def __init__(self, cassandra_client: CassandraClient, table: str):
+        self.cassandra_client = cassandra_client
+        self.table = table
```

Comment:
No keyspace setup needed?  :thinking:

Reply:
The keyspace is set on the Cassandra client :smile_cat: